kano_feedback/WidgetWindow.py

```diff
@@ -139,14 +139,16 @@ def _initialise_window(self):
                                           hexpand=False)
         prompt.get_style_context().add_class('prompt')
         prompt.set_justify(Gtk.Justification.LEFT)
-        prompt.set_alignment(0.1, 0.5)
         prompt.set_size_request(410, -1)
         prompt.set_line_wrap(True)
 
-        prompt_container = Gtk.Table(1, 1, False)
-        prompt_container.attach(prompt, 0, 1, 0, 1, Gtk.AttachOptions.SHRINK | Gtk.AttachOptions.FILL)
+        prompt_align = Gtk.Alignment(xalign=0.5, yalign=0.5)
```

Comment:
My only worry with this is every function for Gtk.Alignment is said to be [depreciated in 3.14](http://lazka.github.io/pgi-docs/#Gtk-3.0/classes/Alignment.html#Gtk.Alignment). I'm guessing this worked for you though?